apps/web/src/index.css

```
@@ -70,48 +75,72 @@
 
   .markdown-body h1, .markdown-body h2, .markdown-body h3, .markdown-body h4 {
-    @apply mt-5 font-serif font-semibold text-foreground;
+    @apply mt-5 font-serif font-semibold tracking-tight text-foreground;
   }
   .markdown-body h1 { @apply text-xl; }
   .markdown-body h2 { @apply text-base; }
   .markdown-body h3, .markdown-body h4 { @apply text-sm; }
 
-  .markdown-body p, .markdown-body ul, .markdown-body ol,
-  .markdown-body table, .markdown-body blockquote {
+  .markdown-body p, .markdown-body ul, .markdown-body ol,\n  .markdown-body table, .markdown-body blockquote {
```

Stylelint (17.14.0)

[error] Unknown type selector "n" (selector-type-no-unknown)

(selector-type-no-unknown)